src/winter_onboarding_2021/fleet_management_service/routes.clj

```diff
@@ -0,0 +1,9 @@
+(ns winter-onboarding-2021.fleet-management-service.routes
+  (:require [bidi.ring :as br]
+            [ring.middleware.json :refer [wrap-json-response]]
+            [winter-onboarding-2021.fleet-management-service.handler :as handler]))
+
+(def routes [["/cabs"] handler/fetch-all-cabs])
```

Comment:
Instead of a dummy `/cabs` endpoint, you could add a health check endpoint. The health check endpoint would be useful for the CI/CD setup as well.

Reply:
Done